packages/e2e/cypress/e2e/api/projectPermissions.cy.ts

```diff
@@ -456,9 +491,169 @@ describe('Lightdash API tests for member user with editor project permissions',
     });
 });
 
-describe('Lightdash API tests for member user with viewer project permissions', () => {
+describe('Lightdash API tests for member user with interactive_viewer project permissions', () => {
     let email;
+
+    before(() => {
+        cy.loginWithPermissions('member', [
+            {
+                role: 'interactive_viewer',
+                projectUuid: SEED_PROJECT.project_uuid,
+            },
+        ]).then((e) => {
+            email = e;
+        });
+    });
     beforeEach(() => {
+        cy.loginWithEmail(email);
+    });
+    it('Should identify user', () => {
+        cy.request(`${apiUrl}/user`).then((resp) => {
+            expect(resp.status).to.eq(200);
+            expect(resp.body.results).to.have.property('email', email);
+            expect(resp.body.results).to.have.property('role', 'member');
+        });
+    });
+
+    it('Should get success response (200) from POST runQuery', () => {
+        const projectUuid = SEED_PROJECT.project_uuid;
+
+        const endpoint = `/projects/${projectUuid}/explores/customers/runQuery`;
+        cy.request({
+            url: `${apiUrl}${endpoint}`,
+            headers: { 'Content-type': 'application/json' },
+            method: 'POST',
+            body: runqueryBody,
+        }).then((resp) => {
+            expect(resp.status).to.eq(200);
+            expect(resp.body).to.have.property('status', 'ok');
+        });
+    });
+    it('Should get success response (200) from POST downloadCsv', () => {
+        const projectUuid = SEED_PROJECT.project_uuid;
+
+        const endpoint = `/projects/${projectUuid}/explores/customers/downloadCsv`;
+        cy.request({
+            url: `${apiUrl}${endpoint}`,
+            headers: { 'Content-type': 'application/json' },
+            method: 'POST',
+            body: runqueryBody,
+        }).then((resp) => {
+            expect(resp.status).to.eq(200);
+            expect(resp.body).to.have.property('status', 'ok');
+        });
+    });
+    it('Should get success response (200) from POST runUnderlyingDataQuery', () => {
+        const projectUuid = SEED_PROJECT.project_uuid;
+
+        const endpoint = `/projects/${projectUuid}/explores/customers/runUnderlyingDataQuery`;
+        cy.request({
+            url: `${apiUrl}${endpoint}`,
+            headers: { 'Content-type': 'application/json' },
+            method: 'POST',
+            body: runqueryBody,
+        }).then((resp) => {
+            expect(resp.status).to.eq(200);
+            expect(resp.body).to.have.property('status', 'ok');
+        });
+    });
+    it('Should get success response (200) from POST runDashboardTileQuery', () => {
+        const projectUuid = SEED_PROJECT.project_uuid;
+
+        const endpoint = `/projects/${projectUuid}/explores/customers/runDashboardTileQuery`;
```

Comment:
This is the problematic endpoint, `viewers` can do this but they can pass the exact `runQuery` body, so as an API consumer I can do the same thing an `interactive_viewer` can

Reply:
Yeah, see my other comment